internal/util/crypto.go

```diff
@@ -46,6 +47,14 @@ const (
 	// Passphrase size - 20 bytes is 160 bits to satisfy:
 	// https://tools.ietf.org/html/rfc6749#section-10.10
 	encryptionPassphraseSize = 20
+	// podNamespace ENV should be set in the cephcsi container
+	podNamespace = "POD_NAMESPACE"
+
+	// kmsConfigmapName env to read a configmap by name
+	kmsConfigmapName = "KMS_CONFIGMAP_NAME"
+
+	// defaultConfigMapToRead default configmap name to fetch kms connection details
+	defaultConfigMapToRead = "csi-kms-connection-details"
```

Comment:
- `kmsConfigmapName` with **Configmap**
- `defaultConfigMapToRead` with **ConfigMap**

As it is about a Kubernetes ConfigMap, use **ConfigMap** for both please